src/game/AI/BaseAI/PetAI.cpp

```diff
@@ -40,17 +40,9 @@ PetAI::PetAI(Creature* creature) : UnitAI(creature), m_creature(creature), inCom
     m_AllySet.clear();
     UpdateAllies();
 
-    switch (((Pet*)creature)->getPetType())
-    {
-        case HUNTER_PET:    //hunter pets attack from behind
-            m_attackAngle = M_PI_F;
-            break;
-        case MINI_PET:
-            SetReactState(REACT_PASSIVE);
-            break;
-        default:
-            break;
-    }
+    //hunter pets attack from behind
```

Comment:
This thing here is wrong for classic, you mentioned something about that @Warlockbugs ? it should be removed entirely then?

Reply:
https://wow.gamepedia.com/Patch_2.3.0#General
> Pets will try to get behind their targets when engaging in melee combat.

We can comment this functionality out for the time being and later put under config value which would default to disabled for vanilla. This kind of QoL is purely server-side and can be very useful, simply removing feels like a waste.